checkov/main.py

```diff
@@ -654,6 +657,10 @@ def add_parser_args(parser: ArgumentParser) -> None:
                default=False,
                action='store_true',
                help='Enable combine tf graph and rf plan graph')
+    parser.add('--no-fail-on-crash',
+               default=False,
+               action='store_true',
+               help='Return exit code 0 instead of 2')
```

Comment:
```suggestion
    parser.add('--no-fail-on-crash',
               default=False,
               env_var='CHECKOV_NO_FAIL_ON_CRASH',
               action='store_true',
               help='Return exit code 0 instead of 2')
```